src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -165,8 +165,10 @@ class SelectClauseSegment(ansi_dialect.get_segment("SelectClauseSegment")):  # t
             Sequence(
                 "AS",
                 "STRUCT",
-                Ref("StarSegment"),
-                Ref("StarModifierSegment", optional=True),
```

Comment:
Oddly, this segment is not defined anywhere, and apparently never was. 🤯 I wonder why this isn't failing. Maybe because it's optional?